### DISCUSSION TITLE
Unexpected Disconnection

### DISCUSSION THREAD

**Discussion by angelmarlo** [arkypita/LaserGRBL#2035] [Q&A]
my Laser keeps stoping in the middle of engraving and lasergrbl pops up with a message that says Unexpected Disconnection i have windows 10 updated with 4 gb ram the usb cord is brand new and came with the laser so its not that. Whats going on?

**Comment by angelmarlo** [comment]
![screen shot](https://user-images.githubusercontent.com/127887099/225107404-aa78d005-6c3e-4512-90a7-4e51535e4207.png)

**Comment by Ronald1971258** [comment]
No meu laser Sculpfun S9 também acontece isto; notei que é de interferência na rede elétrica. Quando ligo o exaustor da sala do laser dá um pulso na rede elétrica e a maquina perde a conexão da porta USB. Tentei por um estabilizador de voltagem e aterrar maquina/computador e estabilizador...não adiantou.  Não sei mais o que tentar fazer; estou perdendo muitos projetos.
